apps/web/src/api/api.client.ts

```diff
@@ -7,6 +7,7 @@ interface IOptions {
   absoluteUrl: boolean;
 }
 
+// @deprecated Migrate all api methods to the new buildAPIHTTPClient that allows runtime configuration on the client object.
```

Comment:
This post-release clean-up work needs to happen to allow us to have way more predictable studio components that use a single source of hooks to speak to bridge or Novu API.